src/nvim/syntax.c

```diff
@@ -7262,6 +7265,28 @@ int syn_get_final_id(int hl_id)
   return hl_id;
 }
 
+/* Refresh the color attributes of all highlight
+ * groups.
+ * This usually needs to be done after the "Normal"
+ * group is modified to update the groups that might
+ * be using "bg" or "fg".
+ */
+void highlight_groups_refresh(void)
```

Comment:
also mark `static`. reason I suggested `highlight_groups_refresh` name was because I was suggesting to put the `ui_refresh` call in here as well. But it's fine like this, but a better name then would be `highlight_attr_set_all` (to hint  its relation to `set_hl_attr`).